frontend/src/components/Welcome.vue

```diff
@@ -53,9 +53,9 @@ export default {
   data: () => ({
     documentation: [
       {
-        text: 'Datashare Frontend UI Doc',
+        text: 'Datashare User Guide',
         href:
-          'https://github.com/GoogleCloudPlatform/datashare-toolkit/blob/master/frontend/README.md'
+          'https://github.com/GoogleCloudPlatform/datashare-toolkit/tree/master/frontend/user-guide/README.md'
```

Comment:
We should add a build tag env var to reference the appropriate user guide. I'll add something on the backlog.